packages/arb-avm-cpp/cavm/cexecutioncursor.cpp

```diff
@@ -34,8 +34,12 @@ int executionCursorMachineHash(CExecutionCursor* execution_cursor_ptr,
     auto executionCursor = static_cast<ExecutionCursor*>(execution_cursor_ptr);
     try {
         auto index_result = executionCursor->machineHash();
+        if (!index_result) {
+            // Unable to compute machine hash
+            return false;
+        }
```

Comment:
The execution cursor should never be in a position where this is true, right? Might want to add a `std::cerr` here if that's the case

Reply:
👍